README.md

```diff
@@ -26,6 +26,14 @@ Usage
         -n  specify file descriptor limit
         -V  print the version of SNIProxy and exit
 
+Usage: Docker
+-----
+
+    docker run \
+        -it \
+        --network host \
+        --volume /etc/sniproxy.conf:/etc/sniproxy.conf:ro \
+        moghaddas/sniproxy:latest -c /etc/sniproxy.conf
```

Comment:
This is _your_ own image, not authoritative for this repository, or is it?

Perhaps change the usage to include how to build it and change his line into the image that will be built locally (`sniproxy`).